src/main/java/seedu/duke/InternshipList.java

```diff
@@ -53,4 +55,39 @@ public void listAllInternships() {
             }
         }
     }
+
+    // Method to list all internships in sorted order without modifying the IDs
+    public void listInternshipsSortedByRole() {
+        ArrayList<Internship> sortedList = new ArrayList<>(internships);
```

Comment:
Toby I think the naming of sortedList can be changed, something like sortedInternships might be more appropriate.